README.md

```diff
@@ -1 +1,17 @@
-# esp-rom-elfs
+# ESP32 series ROM ELF files
+
+The [Releases](https://github.com/espressif/esp-rom-elfs/releases) page of this repository is used to publish the ELF files of ROMs used by ESP32 series of chips.
+
+The releases will be updated whenever a new Espressif chip or a chip revision is released.
+
+## Copyrights and License
+
+The ROMs of ESP32 series of chips are Copyright (c) 2015-2022 Espressif Systems (Shanghai) Co. Ltd.
+
+The ROMs also include various third-party libraries. The list along with the respective licenses is available [here](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/COPYRIGHT.html#rom-source-code-copyrights).
+
+The ROM code of the currently available ESP32 series of chips is not open-source.
```

Comment:
```suggestion
The ROM source code of ESP32 series chips is proprietary and confidential.
```

Reply:
While I think the way you put it is technically correct, I think there is a slight difference in the perceived meaning :)

I wanted to say that just for the chips available now, the ROM is not open source. (Implied: we might be able to make it open source for the future chips.)

On the other hand, "the ROM code is proprietary and confidential" sounds like a stubborn position of a chip vendor who has decided that it's the way this will be, forever.

Reply:
After thinking about this for a bit, I have opted to keep it as is. Thanks for the suggestions though!